utils/docker/peer/Dockerfile

```diff
@@ -90,11 +90,13 @@ RUN cd $FABRIC_PATH \
  && git config user.email "FPC" \
  && git am $FPC_PATH/fabric/*patch \
  && git config --unset user.email \
- && GO_TAGS=pluginsenabled make peer orderer cryptogen configtxgen
+ && make orderer cryptogen configtxgen
 
 # Build FPC plugins for peer
 RUN cd $FPC_PATH \
  && make plugins \
+ && make tlcc_enclave \
+ && make tlcc \
```

Comment:
is there a reason to make these individually vs all in one invocation (and maybe with `-j`) ?

Reply:
I made it now all in one invocation; `-j` does not work here as tlcc depends on tlcc_enclave; so order matters here; though plugins and tlcc_enclave could be built in parallel